extensions/amp-user-notification/amp-user-notification.md

```diff
@@ -163,7 +163,40 @@ will be passed in future requests to data-show-if-href)
 If not specified, AMP will not send a request upon dismissal and will only store "dismissed"
 flag for the specified ID locally.
 
----
+**persistent-dismissal** (Optional)
```

Comment:
Almost :) Let's make it `data-persist-dismissal` and then LGTM. I.e. "persistent" -> "persist".